US-CBP/cbp-theme
# ds-css build configuration updates

#### What's this PR do?
Updates the ds-css build configuration and package.json files with additional tools for things like:
- optimization/minification
- watch
- sourceMap

#### Where should the reviewer start?
Can look at the webpack.config.js file and package.json files to see updates

#### How should this be manually tested?
Can do these steps:
1. nuke node_modules
run: rm -rf node_modules

2. reinstall devDependencies
run: npm i

3. create dev build
run: npm run dev

4. Then can review new files created within the dist/ folder for:
index.html
css/*
ds/*

5. Then can run local page on dev server
run: npm run start

6. Use browser dev tools to check source code for files referenced and make sure everything works as expected.

#### Any background context you want to provide?
just continuing the initial build process

#### What are the relevant tickets?
n/a

#### Screenshots (if appropriate)
n/a

#### Questions:
- Is there a blog post?
n/a

- Does the knowledge base need an update?
n/a
